apps/pom.xml

```diff
@@ -75,8 +75,8 @@
         <maven.jacoco.skip>true</maven.jacoco.skip>
         <sonar.java.coveragePlugin>jacoco</sonar.java.coveragePlugin>
         <sonar.dynamicAnalysis>reuseReports</sonar.dynamicAnalysis>
-        <sonar.jacoco.reportPath>${project.build.directory}/coverage-reports/jacoco-ut.exec</sonar.jacoco.reportPath>
-        <sonar.jacoco.itReportPath>${project.build.directory}/coverage-reports/jacoco-it.exec</sonar.jacoco.itReportPath>
+        <sonar.coverage.jacoco.xmlReportPaths>${project.build.directory}/target/coverage-reports/jacoco-ut/jacoco.xml</sonar.coverage.jacoco.xmlReportPaths>
+        <sonar.coverage.jacoco.itReportPath>${project.build.directory}/target/coverage-reports/jacoco-it/jacoco.xml</sonar.coverage.jacoco.itReportPath>
```

Comment:
Lines 78 & 79 - do we need `/target/`? The  plugin config has its `outputDirectories` set to
`<outputDirectory>${project.build.directory}/coverage-reports/jacoco-it</outputDirectory>`
`<outputDirectory>${project.build.directory}/coverage-reports/jacoco-ut</outputDirectory>`

Reply:
Let me run it without and see ...